packages/ripple-nuxt-tide/lib/module.js

```diff
@@ -11,8 +11,22 @@ const nuxtTide = function (moduleOptions) {
     ...this.options.proxy,
     // Set the proxy timeout for requesting to Tide API as 9 seconds.
     // POST request to Tide normally need more than 5 seconds to get response.
-    '/api/v1/': { target: options.baseUrl, proxyTimeout: 9000 },
-    '/sites/default/files/': options.baseUrl
+    '/api/v1/': {
+      target: options.baseUrl,
+      proxyTimeout: 9000,
+      onProxyRes (proxyRes, req, res) {
+        proxyRes.headers['X-SDP-APP-TYPE'] = 'tide'
```

Comment:
@tim-yao , I realised an issue with this. You might need an if before it like this to set the var

        let cacheHeader = 'tide';
        if (response.headers['x-sdp-app-type'] !== undefined) {
          cacheHeader = response.headers['x-sdp-app-type'];
        }
....
        proxyRes.headers['X-SDP-APP-TYPE'] = cacheHeader

Or you might need to move this code down to packages/ripple-nuxt-tide/lib/templates/axios.js, I'm not sure. But this should pull and pass through the X-SDP-APP-TYPE header from the API it's proxying rather than hard code the value, defaulting to 'tide'.

Reply:
Will Drupal ever proxy an API directly? My understanding is that `/api/v1` is set up to proxy Drupal directly. Requests that go to elastic don't go via Drupal they go direct to elastic. So could this ever be anything other than tide?

Reply:
@steveworley that's what I thought, and why I just hard code `tide` here. As my understanding, `/api/v1/` is pointed to our Drupal Tide API, so should always be `tide`.

Any other proxy responses, if they need go through our Nuxt proxy, then we will set the value in each of proxy config, like I did for `/sites/default/files/` below: https://github.com/dpc-sdp/ripple/pull/625/files/843c3f02ed6c85db9db6238fe514fd8681d678ac#diff-2538535bb7172f81b6518c4233db3ca2R27

Say we need proxy `/mailgun-api/` then we will have a separated proxy config for it. And set 'X-SDP-APP-TYPE' to `mailgun`.

@kurtfoster that's what I was talking about before. Any thoughts?